create/how_to/updating.md

```diff
@@ -0,0 +1,231 @@
+# Update to the Latest MeiliSearch Version
+
+Currently, MeiliSearch versions [are not compatible with each other](/learn/getting_started/installation.md#updating-meilisearch). The following guide will walk you through all the steps to migrate your database from an older version of MeiliSearch to the most recent one.
+
+If you already have a MeiliSearch database with some data you don’t want to lose, you are in the right place!
+
+## Verify your database version
+
+Before we begin, you need to verify your database's **expected MeiliSearch version**, i.e. the version that indexed the data. You can do so by launching a MeiliSearch instance:
+
+```bash
+./meilisearch
+```
+
+If MeiliSearch launches successfully, use the [get version endpoint](/reference/api/version.md) and [proceed to the next step](#proceed-according-to-your-database-version).
+
+```bash
+curl -X GET 'http://127.0.0.1:7700/version'
+```
+
+If you get the error `Cannot open database, expected MeiliSearch engine version: X.X.X, current engine version Y.Y.Y`, **your database is not compatible with the currently installed MeiliSearch version**. You need to download the expected version and use that instead.
```

Comment:
I see, I thought it was clear in this location since it's the same language as in the error message. That's a good note, thanks. Sometimes it can be hard to strike a balance between speed/efficiency and clarity...

Reply:
Clarity trumps everything IMNSHO